.pylintrc

```diff
@@ -6,13 +6,14 @@
 # C1001: Old-style class defined.
 # D100:  Missing docstring in public module.
 # E1101: %s %r has no %r member.
+# MC0001: mccabe plugin
```

Comment:
@RishabhJain2018 this file should not be included in this PR? Btw, why do you need this?